test/assembly/data/schema_export_truth.gql

```diff
@@ -306,5 +306,110 @@ transaction sub relation,
     relates transaction_merchandise,
     relates transaction_vendor;
 
+born-in-location-implies-residency sub rulewhen {
```

Comment:
errr where did you get these rules? If the rest of the schema comes from our simulation, we should use those rules. If that's where these were sourced, are they supposed to be `sub rulewhen`? That looks like a bug to me... If this test passes i'm not sure that's correct!!

Reply:
That's not right. There's a println(",") missing after 'rule'. I've fixed that now.